src/index.tsx

```diff
@@ -1,13 +1,31 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { HomePage } from './pages/HomePage';
+import { PageNotFound } from './pages/PageNotFound';
+import { PeoplePage } from './components/PeoplePage';
 
 createRoot(document.getElementById('root') as HTMLDivElement).render(
   <Router>
-    <App />
+    <Routes>
+      <Route path="/" element={<App />}>
```

Comment:
You can move all paths to enum, then use it in all places of your application. (also, in this place)